sushy_oem_idrac/resources/manager/manager.py

```diff
@@ -83,6 +102,90 @@ class DellManagerExtension(oem_base.OEMResourceBase):
     def import_system_configuration_uri(self):
         return self._actions.import_system_configuration.target_uri
 
+    @property
+    def job_service_uri(self):
+        return self._links.job_service.resource_uri
+
+    @property
+    def get_jobs_uri(self):
+        return self._links.jobs.resource_uri
+
+    @property
+    def get_remote_service_api_uri(self):
+        return ('%s%s' %
+                (self._links.lc_service.resource_uri,
+                self.REMOTE_API_STATUS_URI))
+
+    def clear_job_queue(self, job_ids=['JID_CLEARALL']):
+
+        if job_ids is None:
+            return None
+
+        delete_job_queue_uri = '%s%s' % (self.job_service_uri, self.CLEAR_JOBS_URI)
+        for job_id in job_ids:
+            payload = {'JobID':job_id}
+
+            try:
+                delete_job_response = asynchronous.http_call(
+                        self._conn, 'post',
+                        delete_job_queue_uri,
+                        headers = self.HEADERS,
+                        data = payload,
+                        verify = False)
+                return delete_job_response
+
+            except (sushy.exceptions.ServerSideError,
+                    sushy.exceptions.BadRequestError) as exc:
+                LOG.error('Dell OEM clear job queue failed, %s', err_msg)
+                raise sushy.exceptions.ServerSideError(error=exc)
+
+    def reset_idrac(self,manager=None):
+        payload = {"ResetType":"GracefulRestart"}
+
+        try:
+            reset_job_response = asynchronous.http_call(
+                            self._conn, 'post',
+                            manager._actions.reset.target_uri,
+                            data=(payload))
+
+        except (sushy.exceptions.ServerSideError,
+                sushy.exceptions.BadRequestError) as exc:
+            LOG.error('Dell OEM reset idrac failed, Reason : %s', exc)
+            raise sushy.exceptions.ServerSideError(error=exc)
+
+        LOG.info("iDRAC has reset, Waiting for the iDRAC to become ready")
+        if(utils.wait_for_idrac_ready(self) == True):
+            LOG.info("Dell OEM iDRAC reset successfuly")
+            return reset_job_response
+        else:
+            err_msg = 'Timeout reched to become iDRAC ready'
```

Comment:
Can you change the above message to:
'Timed out while waiting for the iDRAC to become ready after reset'

Reply:
Addressed, Added meaningful sentence as suggested.